ic-os/components/hostos-scripts/build-bootstrap-config-image.sh

```diff
@@ -34,9 +34,12 @@ options may be specified:
   --ipv4_gateway a.b.c.d
     (optional) Default IPv4 gateway (e.g. 18.208.190.33).
 
-    --domain domain
+  --domain domain
     (optional) The domain name to assign to the guest.
 
+  --node_remuneration_type node_remuneration_type
+    (optional) The node remuneration type used for node rewards
+
```

Comment:
The reason I left node_remuneration_type optional in build-bootstrap-config-image (and generate-guestos-config), even though it's required in setupOS, is for backwards compatibility. This script runs on reboot, so after a HostOS upgrade on mainnet, node_remuneration_type will not be in config.ini, and therefore the field should be optional.

But note that once we are on the new config, this script as well as generate-guestos-config.sh and bootstrap-ic-node.sh will all be removed.

Reply:
I totally agree with that. We should make it non-optional only in SetupOS!
(and also in the registry at some point, but that shouldn't be a big concern for you)